packages/gatsby-theme-docs/src/overrides/page-header-side.js

```diff
@@ -1,2 +1,23 @@
+import React from 'react';
+import css from '@emotion/css';
+import { designSystem } from '@commercetools-docs/ui-kit';
+import UserResearchBanner from '../components/user-research-banner';
+
 // A React component to be rendered in the top-right corner of a content page
-export default () => null;
+// eslint-disable-next-line react/display-name
+export default () => {
+  return (
+    <div>
+      <div
+        css={css`
+          padding-bottom: ${designSystem.dimensions.spacings.m};
+        `}
+      >
+        <svg viewBox="0 0 16 16" width="24" height="24">
```

Comment:
I also implemented the Github icon so that you can see how the upper right corner behaves with multiple components rendered...but I guess we should remove that before merging.

Reply:
The specific example here should not be directly in the theme but e.g. in the API Smoke Test site as an override so we can test and compare two sites with and without the override.   In the smoke test site it should actually stay (you can use a different Icon but it would stay as a regression test)